PyRoute/Calculation/TradeCalculation.py

```diff
@@ -409,9 +456,50 @@ def route_weight(self, star, target):
             target) + " must be positive"
         return weight
 
+
     def unilateral_filter(self, star):
         if star.zone in ['R', 'F']:
             return True
         if star.tradeCode.barren:
             return True
         return False
+
+    def is_sector_trade_balanced(self):
```

Comment:
Shouldn't there also be an `is_allegiance_trade_balanced()` and `is_allegiance_pass_balanced()` methods?

Reply:
By symmetry, yes